index.html

```diff
@@ -28,14 +28,18 @@
             </header>
             <main>
                 <h1>Welcome to Ryan's CSD120 Website</h1>
-                    <img src="IMG_5122.jpg" alt="Photo of myself" height="400px" width="300px"></br>
+                    <a href="images/IMG_5122.jpg">
+                        <img src="images/IMG_5122.jpg" alt="Photo of myself" height="400px" width="300px"
+                            sizes="(max-width:500px) 150px, 300px"
+                            srcset="images/IMG_5122_150w.jpg 150w, images/IMG5122_300w.jpg 300w">
+                    </a>

```

Comment:
Missing hyperlink
Missing srcset and sizes attributes
NOTE: I just made up filenames for images because you haven't added your images to your git repo yet.